src/SFA.DAS.AdminService.Web/wwwroot/stylesheets/application.css

```diff
@@ -1657,3 +1657,49 @@ TODO: change to 'das-page-navigation' and BEM
 .govuk-textarea--error-override {
   border: 4px solid #b10e1e;
 }
+
+.das-button--inline-link {
+  font-family: "nta", Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  color: #1d70b8;
+  text-decoration: underline;
+  font-size: inherit;
+  font-weight: inherit;
+  background: none;
+  border: none;
+  padding: 0;
+  margin: 0;
+  cursor: pointer;
+  text-align: left;
+  display: inline;
+  line-height: 1;
+}
+
+.das-button--inline-link:focus {
+  outline: 3px solid #ffbf47;
+  outline-offset: 0;
+  background-color: #ffbf47;
+}
+
+.das-button--inline-link:link {
+  color: #1d70b8;
+}
+
+.das-button--inline-link:visited {
+  color: #4c2c92;
+}
+
+.das-button--inline-link:hover {
+  color: #2b8cc4;
+}
+
+.das-button--inline-link:active {
+  color: #0b0c0c;
+  background-color: #ffbf47;
+}
+
+.das-button--inline-link:focus {
+  color: #0b0c0c;
+  background-color: #ffbf47;
+}
```

Comment:
Are these to be separate from those in lines 1679 above?

Reply:
No reason for them to be. I'll put them into one rule.